### DISCUSSION TITLE
What's your favorite feature of the glance widget on Android devices?

### DISCUSSION THREAD

**Discussion by viralsweet** [community/community#144534] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm curious to know what you all love most about the Glance widget on Android. For me, having quick access to news and updates without unlocking my phone is super handy, but I’m sure there are other cool features I haven’t explored yet. Whether it’s the sports updates, little games, or just the convenience, what’s your favorite feature? Would love to hear your thoughts!

**Comment by samus-aran** [comment]
Hi @viralsweet ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this.... like this glance widget you're so fond of. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by viralsweet** [reply]
Got it
